tests/integration_test/grpc_integration_test.go

```diff
@@ -370,24 +370,24 @@ func TestGRPCJwt(t *testing.T) {
 			clientProtocol:   "grpc",
 			method:           "ListShelves",
 			token:            testdata.FakeCloudToken,
-			wantError:        "code = PermissionDenied desc = Audiences in Jwt are not allowed",
-			wantGRPCWebError: "403 Forbidden, Audiences in Jwt are not allowed",
+			wantError:        `code = PermissionDenied, message = {"code":403,"message":"Audiences in Jwt are not allowed"}`,
+			wantGRPCWebError: `403 Forbidden, {"code":403,"message":"Audiences in Jwt are not allowed"}`,
```

Comment:
Is this error from gprc-status response header.   If no, could you add some negative tests to verify error message from grpc-message and grpc-status

Reply:
Yes, `code = ` is from grpc-status and `message=` from grpc-message.  I translate the plain error in golang grpc to rpc.Status in bookstore client,